include/deal.II/matrix_free/task_info.h

```diff
@@ -496,6 +501,36 @@ namespace internal
        */
       std::vector<unsigned int> ghost_face_partition_data;
 
+      /**
+       * TODO
```

Comment:
Can you please document this member (and the others below)?

Reply:
Done!